### PR TITLE
Added full support balanced quotation marks

### PR DESCRIPTION
**Pull request by PanderMusubi**
Closing https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1043 and https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1038

### CODE REVIEW

File: app/src/main/res/xml-ar/ext_kbd_bottom_row_regular.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
         <Key android:codes="32" android:keyWidth="30%p"/>
 
         <Key android:codes="46" android:keyLabel="." android:popupCharacters=";:-_\u00b7\u2026"/>
-        <Key android:codes="39" android:keyLabel="\'" android:popupCharacters="&#34;\u201e\u201d"/>
+        <Key android:codes="39" android:keyLabel="\'" android:popupCharacters="&#34;\u201c\u201c\u201e\u201d\u2018\u2019"/>
```
Comment (menny) [inline app/src/main/res/xml-ar/ext_kbd_bottom_row_regular.xml]: Is this the same character twice?

Comment (PanderMusubi) [inline app/src/main/res/xml-ar/ext_kbd_bottom_row_regular.xml]: that is a typo, sorry. only there or on more places?

Comment (PanderMusubi) [inline app/src/main/res/xml-ar/ext_kbd_bottom_row_regular.xml]: Think you can use a string from values.xml there.

**Comment by menny** [comment]
Want to give that a try?

**Comment by PanderMusubi** [comment]
not particularly as keyboards, especially in the language packs, are a lot of custom work and looking up everything twice might complicate flexibility in development

**Comment by PanderMusubi** [comment]
What would be a nice feature for developers, is to generate a pdf anf/or html and/or markup with an over for each keyboard but with all the popup characters. For example:

**`q`**|**`w`**|**`e`**|**`r`**|**`t`**
`1`|`2`|`3`|`4`|`5` `€`

**`a`**|**`s`**|**`d`**|**`f`**|**`g`**
`á` `à` `ä` `â` `å`|`ß`|`đ`| |

but than neatly lined out of course.

**Comment by menny** [comment]
@PanderMusubi a couple of things here:
1) remove the extra `\u201c`
2) rebase your PR and force push

**Pull request merged**